__tests__/app.test.js

```diff
@@ -44,3 +44,52 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  it("200 - should respond with correct status code and contain correct feilds", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((res) => {
+        const {
+          data: [review],
+        } = res.body;
+
+        expect(review).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
Need to check the id is the requested one you've asked for - in this case 1. 

At the moment, this would pass for every object sent back

Reply:
line 58 could replace line 61